hugo/content/blog/2022-01-16-mumble-1-4-230.md

```diff
@@ -0,0 +1,392 @@
+1.4.
+
+![Access restricted channels](/blog/media/AccessRestrictedChannels.png)
+
+
+### Markdown
+
+For those of you who don't know what markdown is, we recommend a quick read of the respective [Wikipedia
+article](https://en.wikipedia.org/wiki/Markdown). In simple words it allows formatting of text by simply typing a few extra characters in it.
+
+Mumble now supports markdown for text messages written in the normal chat bar. It is not supported for cases in which you use the dedicated message
+editor (e.g. for private messages). Furthermore only a subset of markdown is supported: sections, inline-code, code-blocks, bold, italic and
+strike-through text, links and quotes. _Not_ supported are lists and tables.
+
+This feature is probably best explained by an example. Try pasting the following text into the chat bar and see for yourself:
+
+````
+# Markdown
+## Code
+We can have `inline` code and
+```
+code
+    blocks
+```
+
+## Emphasis
+**bold** and *italic* (__bold__ and _italic_ also works) text is supported. ~~Scratch that~~
+
+## Links
+We can have [links](google.de) and www.google.de
+
+## Quotes
+> I am a quote
+Although these are not pretty, they do work and are supported.
+
+## Escaping
+We can escape special characters using the \\ character: \*not italic\* and \*\*not bold\*\*
+````
+
+![Markdown](/blog/media/Markdown.png)
+
+
+### Stereo playback
+
+With version 1.4 Mumble (finally) learned how to play back stereo audio streams. This means that it is now possible to send a stereo stream to a
+Mumble client and it will actually be played back without being mixed down to a mono stream.
+
+Note however that this feature is currently restricted to playback. The official Mumble client will still continue sending audio as mono. This feature
+is primarily intended to be made use of by (music) bots.
+
+If you want to try this feature out yourself, you can either write a bot using [pymumble](https://github.com/azlux/pymumble) version 1.2 or later or
+you can simply use a recent version of [botamusqiue](https://github.com/azlux/botamusique).
+
+
+### Nicknames
+
+For those of you who are annoyed by people using excessively long or complicated names or people changing their nickname frequently, Mumble 1.4
+introduces the possibility to set a nickname for users.
+
+Depending on your configuration, a nickname will either be displayed next to a user's name or replace it entirely.
+
+The nickname will persist across restarts of client and server. In fact it'll last until you either remove it again or the person the nickname was
+assigned to changes their client-certificate (Mumble's way of keeping track of a user's identity).
+
+Note that nicknames are _not_ tied to a particular server. That means that if you assign a nickname to a user on server A and then you meet that
+person again on server B, they will still have that very same nickname set on server B.
```

Comment:
We could mention that this is like it was before for users added to the friend list. For example:

This behavior is the same as the friend list feature. After adding a user as a friend on server A, you could identify them on server B by their known nickname even when they use a different nickname on that server B.